poet/sawtooth_poet/validator_registry/processor/handler.py

```diff
@@ -0,0 +1,337 @@
+        raise InternalError('Unable to get {}'.format(address))
+
+    if len(entries_list) != 0:
+        validator_info.ParseFromString(entries_list[0].data)
+
+    return validator_info
+
+
+def _add_validator_to_validator_map(state, validator_id, anti_sybil_id):
+    validator_map = _get_validator_map(state)
+    if anti_sybil_id not in validator_map.validator_ids:
+        validator_map.validator_ids[anti_sybil_id] = validator_id
+        address = "6a4372" + hashlib.sha256(
+            'validator_map'.encode()).hexdigest()
+        try:
+            addresses = list(state.set(
+                [StateEntry(address=address,
+                            data=validator_map.SerializeToString())],
+                timeout=STATE_TIMEOUT_SEC)
+            )
+
+        except FutureTimeoutError:
+            LOGGER.warning(
+                'Timeout occured on state.set([%s, <value>])', address)
+            raise InternalError(
+                'Unable to add {} to valdiator_list'.format(validator_id))
+
+        if len(addresses) != 1:
+            LOGGER.warning(
+                'Failed to save value on address %s', address)
+            raise InternalError(
+                'Unable to add {} to validator_map'.format(validator_id))
+
+        LOGGER.info("Validator id %s was added to the validator_map",
+                    validator_id)
+
+    else:
+        validator_info = _get_validator_info(state, validator_id)
+
+        validator_info.registered = "revoked"
+        validator_info = validator_info.SerializeToString()
+        _set_validator_info(state, validator_id, validator_info)
+
+
+def _set_validator_info(state, validator_id, validator_info):
+    address = "6a4372" + hashlib.sha256(validator_id.encode()).hexdigest()
+    try:
+        addresses = list(state.set(
+            [StateEntry(address=address,
+                        data=validator_info)],
+            timeout=STATE_TIMEOUT_SEC)
+        )
+    except FutureTimeoutError:
+        LOGGER.warning(
+            'Timeout occured on state.set([%s, <value>])', address)
+        raise InternalError(
+            'Unable to set validator info for {}'.format(validator_id))
+
+    if len(addresses) != 1:
+        LOGGER.warning(
+            'Failed to save value on address %s', address)
+        raise InternalError(
+            'Unable to save validator info for {}'.format(validator_id))
+
+    LOGGER.info("Validator info was set for %s", validator_id)
+
+
+class ValidatorRegistryTransactionHandler(object):
+
+    __REPORT_PRIVATE_KEY_WIF = \
+        '5Jz5Kaiy3kCiHE537uXcQnJuiNJshf2bZZn43CrALMGoCd3zRuo'
+
+    def __init__(self, namespace_prefix):
+        self._report_private_key = \
+            signing.encode_privkey(
+                signing.decode_privkey(self.__REPORT_PRIVATE_KEY_WIF, 'wif'),
+                'hex')
+
+        self._report_public_key = signing.generate_pubkey(
+            self._report_private_key)
+
+        self._namespace_prefix = namespace_prefix
```

Comment:
If the namespace is a module level constant, you don't need this.  Really there's a disconnect between this being a parameter but never actually being used to compute addresses, which could cause a mismatch later down the road.